### DISCUSSION TITLE
네이버로 넘어가는대신 카카오톡api로 바로신청하는건에대하여..

### DISCUSSION THREAD

**Discussion by kikikikikinanana** [changdoc/naver-vaccine-macro#187] [Ideas]
orgcode를공유하는 네이버로 넘어가는게아니라 처음부터 카카오톡계정으로 로그인해서 카카오톡api로 바로신청하는방법은어떤가요? 속도면에서는 카카오톡으로 바로신청넣는게빠를것같은데 안정성에서는 네이버로넘기는게낫나.. 고민이네요

**Comment by changdoc** [comment]
카카오 쪽 신청 페이지로 접근해서 신청도 가능할 것 같습니다만 아직은 저는 계획에 없긴 하네요.
